src/node_platform.h

```diff
@@ -0,0 +1,42 @@
+#ifndef SRC_NODE_PLATFORM_H_
+#define SRC_NODE_PLATFORM_H_
+
+#include "libplatform/libplatform.h"
+#include "node_mutex.h"
+#include "uv.h"
+
+#include <queue>
+
+namespace node {
+
+using v8::Isolate;
+using v8::Platform;
+using v8::Task;
```

Comment:
Can you move these to the .cc file and use fully qualified names here? Rule of thumb: no using declarations in headers.